### PR TITLE
{bio}[GCC/6.4.0-2.28,iccifort/2017.4.196-GCC-6.4.0-2.28] PLINKSEQ v0.10

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/9bf81f5e4818c346a1ea7c2b136da442 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/331ff18125b2d022d0b57102cdbf74ff for a full test report.

File: easybuild/easyconfigs/p/PLINKSEQ/PLINKSEQ-0.10-GCC-6.4.0-2.28.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PLINKSEQ/PLINKSEQ-0.10-GCC-6.4.0-2.28.eb]: @vanzod Please reinstate the attribution from the previous version.

File: easybuild/easyconfigs/p/PLINKSEQ/PLINKSEQ-0.10-iccifort-2017.4.196-GCC-6.4.0-2.28.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PLINKSEQ/PLINKSEQ-0.10-iccifort-2017.4.196-GCC-6.4.0-2.28.eb]: @vanzod Same here - missing attribution.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 0 out of 0 (2 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5837336e3f1d23f73cf9653c939cc1d7 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @vanzod!

**Pull request merged**